Umbraco-Heartcore/Getting-Started-Cloud/GraphQL-Playground/index.md

```diff
@@ -0,0 +1,95 @@
+---
+versionFrom: 8.0.0
+meta.Title: "Umbraco Heartcore GraphQL Playground"
+meta.Description: "Documentation for Umbraco Heartcore GraphQL Playground"
+---
+
+# GraphQL Playground
+
+With GraphQL query language, you will be able to limit the amount of data transferred to and between your Heartcore instance and the client by specifying exactly what kind of information you would like to get in your API calls.
+
+You can test and build GraphQL queries directly in the backoffice-integrated GraphQL Playground.
+
+In the Settings section in the Umbraco Backoffice, you will find the Headless tree. From there you can use the GraphQL Playground to test your queries against your project's schema.
+
+The Playground comes with basic features such as real-time error highlighting, syntax highlighting, formatting, query history and more.
+
+## Accessing GraphQL Playground
+
+1. Log into the project backoffice
+2. Navigate to the Settings dashboard
+3. Find the Headless tree near the bottom of the navigation list
+4. Click on GraphQL Playground
+
+![Where to find GraphQL Playground](images/where-to-find-graphql-playground.png)
+
+From there, you can start querying as soon as you have some content set up.
+
+## Features
+
+### Documentation
+
+GraphQL Playground's built-in documentation can be found on the right side, under the "Docs" tab.
+
+The documentation tab grants a quick overview of how content is structured using the Document Types in your solution. It also gives an idea on how to access specific properties each Document Type has, as well as their properties.
+
+![Documentation Tab in the Playground](images/docs-helper.png)
+
+The documentation will be automatically updated as you add new Document Types to match the schema you have on your project.
+
+## Schema
+
+The Schema tab also can be found on the right side of the GraphQL Playground. It is located below the "Docs" tab.
+
+The Schema menu explains the relationships between specific objets and properties in your solution, and will be updated automatically as you make changes to the project's schema - Document Types, Data Types, and so on.
```

Comment:
make changes to the project's schema meaning Document Types, Data Types, Document Type Compositions and Document Type Elements.